### PR TITLE
Edit UI

### LINKED ISSUE #8: As a user, I want to be able to view the notes/special remarks of a seller
so that I know what to keep in mind when negotiating the price with a potential buyer.

### LINKED ISSUE #144: As a user, I want to be able to identify expired property and appointment listings
so that I can focus on current properties and appointments.

### LINKED ISSUE #145: Add LightTheme.css to change design scheme of Ui
Change design scheme of Ui to light coloured theme.

### LINKED ISSUE #146: Update PropertyCard.java and PropertyListCard.fxml to display Client details as separate Labels in Ui
Update display of client details into separate Labels in Ui:
clientAskingPrice
clientName
clientContact
clientEmail

### LINKED ISSUE #147: Update PropertyCard.java and AppointmentCard.java to grey out HBox of expired Properties and Appointments in Ui
Grey out Property and Appointment listings with expired deadlines or date/time in Ui

### LINKED ISSUE #316: As a user, I want to view all properties and appointments side by side
so that I can simultaneously view related property and appointment data .

### PR DESCRIPTION
**Pull request by candyhy**
Add LightTheme.css to change design scheme of Ui
Update MainWindow.fxml to adopt LightTheme.css
Update PropertyCard.java and PropertyListCard.fxml to display Client details as separate Labels in Ui
Update PropertyCard.java and AppointmentCard.java to grey out HBox of expired Properties and Appointments in Ui

### CODE REVIEW

File: src/main/java/seedu/address/ui/UiManager.java [hunk 1]
```diff
@@ -66,7 +66,7 @@ void showAlertDialogAndWait(Alert.AlertType type, String title, String headerTex
     private static void showAlertDialogAndWait(Stage owner, AlertType type, String title, String headerText,
                                                String contentText) {
         final Alert alert = new Alert(type);
-        alert.getDialogPane().getStylesheets().add("view/DarkTheme.css");
+        alert.getDialogPane().getStylesheets().add("view/LightTheme.css");
```
Comment (w-yuchen) [inline src/main/java/seedu/address/ui/UiManager.java:69]: Will the user be able to change back to DarkTheme?

File: src/main/java/seedu/address/ui/AppointmentCard.java [hunk 2]
```diff
@@ -47,6 +52,15 @@ public AppointmentCard(Appointment appointment, int displayedIndex) {
         remarks.setText(appointment.getRemarks().remark);
         date.setText(appointment.getDate().toString());
         time.setText(appointment.getTime().toString());
+        Date currentDate = new Date(LocalDate.now());
+        Time currentTime = new Time(LocalTime.now());
+
+        if (currentDate.compareTo(appointment.getDate()) > 0) {
+            cardPane.setStyle("-fx-background-color: #696969");
```
Comment (dvdweien) [inline src/main/java/seedu/address/ui/AppointmentCard.java:59]: i like the colour you chose

File: src/main/java/seedu/address/ui/UiManager.java (hunk 1 above)
Comment (candyhy) [reply src/main/java/seedu/address/ui/UiManager.java:69]: Nope, if you guys are agreeable with the new LightTheme Ui, I will go ahead and delete DarkTheme.css

Screenshot 2021-03-24 at 7 00 43 PM

Comment (dvdweien) [reply src/main/java/seedu/address/ui/UiManager.java:69]: izzit possible to have something to toggle between light and dark themes?

Comment (candyhy) [reply src/main/java/seedu/address/ui/UiManager.java:69]: I would say it's not impossible but the purpose of adopting a light theme was to allow the greying out of expired property and appointments to be somewhat observable yet inconspicuous. The grey effect in the already Black/Grey DarkTheme would probably not achieve the same effect. However, if you guys are keen on a dark theme, I can consider a DarkTheme design in a subsequent update- maybe keep the DarkTheme.css file for now

Comment (dvdweien) [reply src/main/java/seedu/address/ui/UiManager.java:69]: i see

**Review by dvdweien — APPROVED** [review]
LGTM

**Pull request merged**